src/render.py

```diff
@@ -1,27 +1,49 @@
 import os
 from constant import HALF_BOARD_SIZE
 
+
 def get_marble_character(value):
     return "◯" if value == 0 else "◉"
-    
-def print_board(board, clear_before_printing = True):
-    
+
+"""
+board = 2d array
+mode = "coords" : display X & Y coordonates on the header and on the left side.
+mode = "rotation" : display rotation input to indicate which enter to rotate each quarters
+clear_before_printing: If true, clear the console before printing the board.
+"""
+def print_board(board, mode="coords", clear_before_printing=True):
```

Comment:
This one is getting hard to read. I wonder if a bit of code duplication wouldn't be better than too much smart code?